src/rgw/rgw_op.cc

```diff
@@ -6545,6 +6518,27 @@ void RGWDeleteMultiObj::execute()
 	      continue;
       }
     }
+
+    // verify_object_lock
+    bool check_obj_lock = obj->have_instance() && bucket->get_info().obj_lock_enabled();
+    if (check_obj_lock) {
+      int get_attrs_response = obj->get_obj_attrs(s->obj_ctx, s->yield);
+      if (get_attrs_response < 0) {
+        if (get_attrs_response == -ENOENT) {
+          // object maybe delete_marker, skip check_obj_lock
+          check_obj_lock = false;
+        } else {
+          // Something went wrong.
+          send_partial_response(*iter, false, "", get_attrs_response);
+          continue;
+        }
+      }
+    }
+    int object_lock_response = verify_object_lock(this, obj->get_attrs(), false, false);
```

Comment:
if this function is really going to take two booleans as arguments, maybe /* */ comment them where it's called

Reply:
This was changed in a later commit.